cli/command/container/tty.go

```diff
@@ -34,18 +34,35 @@ func resizeTtyTo(ctx context.Context, client client.ContainerAPIClient, id strin
 	}
 
 	if err != nil {
-		logrus.Debugf("Error resize: %s", err)
+		logrus.Debugf("Error resize: %s\r", err)
 	}
+	return err
 }
 
 // MonitorTtySize updates the container tty size when the terminal tty changes size
 func MonitorTtySize(ctx context.Context, cli command.Cli, id string, isExec bool) error {
-	resizeTty := func() {
+	resizeTty := func() error {
 		height, width := cli.Out().GetTtySize()
-		resizeTtyTo(ctx, cli.Client(), id, height, width, isExec)
+		return resizeTtyTo(ctx, cli.Client(), id, height, width, isExec)
 	}
 
-	resizeTty()
+	err := resizeTty()
+	if err != nil {
+		retryTimes := 5
+		retry := 0
+		go func() {
+			for {
+				time.Sleep(10 * time.Millisecond)
+				err = resizeTty()
+				if err != nil && retry < retryTimes {
+					retry++
+					logrus.Debugf("Resize tty has retried %d times now\r", retry)
```

Comment:
This will be a bit noisy; also, since this is the CLI, I think this will be printed on `stdout`  / `stderr`, not logged (I think).

I think we should either remove the debug, or only print a debug message if it failed to do the resize after `retryTimes`.

Reply:
Thanks. I also add a small test case.